script/clear_external_cache.pl

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use MusicBrainz::Server::Context;
+
+# This script can be used to clear everything in memcached. It assumes that
+# the CACHE_MANAGER_OPTIONS DBDefs setting resembles the default one (so it
+# has an "external" profile pointing to Cache::Memcached::Fast, and
+# default_profile is set to "external").
+
+my $c = MusicBrainz::Server::Context->create_script_context;
+$c->cache->flush_all;
```

Comment:
Doesn't this delete all the keys from memcached?? What if memcached is shared between multiple applications?? Maybe, a better approach would be to only delete the keys in MusicBrainz' namespace, as defined in `DBDefs -> MEMCACHED_NAMESPACE`.